plugins/parsers/xpath/protocolbuffer_document.go

```diff
@@ -138,7 +138,6 @@ func (d *protobufDocument) GetNodePath(node, relativeTo dataNode, sep string) st
 		if n.Parent != nil && n.Parent.Value() != nil {
 			kind = reflect.TypeOf(n.Parent.Value()).Kind()
 		}
-		fmt.Printf("node %q: %v (value=%v)   parent: %v\n", n.Data, kind, n.Value(), n.Parent.Value())
```

Comment:
Are these helpful? should they be log messages? Same question in xpath/parsers.go

Reply:
I'd love to have those at the debug level.